ui/views.py

```diff
@@ -168,7 +168,7 @@ def extra_context(self):
                 "repo": self.repo,
                 "perms_on_cur_repo": get_perms(self.request.user, self.repo),
                 "vocabularies": {
-                    k: v
+                    k: [x + (x[0].replace("_", " "),) for x in v]
```

Comment:
This replaces `("slug", "count")` with `("slug", "count", "pretty name")`.

For example, `("very_difficult", "3")` would become `("very_difficult", "3", "very difficult")`.